src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state.status,
+  },
+});
+
```

Comment:
- [ Required ] I think it would be better if you also export the action in `categoriesSlice` so that you can access it in your react component. Please consider exporting the action as shown below.

```JavaScript
  export const { checkStatus } = categoriesSlice.actions
```

Reply:
Added the code as suggested.